### DISCUSSION TITLE
Alerts tab not remembering sorting

### DISCUSSION THREAD

**Discussion by Jacob-Tate** [Security-Onion-Solutions/securityonion#8091] [General] [answered]
I just updated to 2.3.130 and found that the Alerts tab keeps forgetting my sorting settings.

For example in the prior release when I say selected to sort by `event.severity_label` it was remembering that when I refresh but with the new update it appears to forget that.

To recreate all I have to do is sort by `event.severity_label` and select refresh

**Comment by dougburks** [comment, marked as answer]
We'll take a look as time allows.

**Reply by dougburks** [reply]
This will be fixed in the upcoming 2.3.140 release:
https://github.com/Security-Onion-Solutions/securityonion/issues/8133
